packages/marketplace/package.json

```diff
@@ -79,7 +79,8 @@
     "@openzeppelin/contracts": "^4.9.2",
     "@openzeppelin/contracts-upgradeable": "^4.9.2",
     "@openzeppelin/hardhat-upgrades": "^2.2.1",
+    "@sandbox-smart-contracts/asset": "1.0.2",
```

Comment:
Where are we using asset?

Reply:
we aren't, but the jira tocket that was assigned to me is titled "Fix the version in the marketplace package according to Asset input" with the description "What Woj did was to add the missing Asset dependencies so that it can be imported to the marketplace package." So I figured I should add it.